### PR TITLE
Exclude invalid area codes

### PR DESCRIPTION
**Pull request by kevintraver**

### CODE REVIEW

**Comment by rstacruz** [comment]
Thanks fro the contribution, but I'm not sure it's a good idea to implement it this way... you will be doing `Range#to_a` of a pretty big range every time you call the function.

**Comment by kevintraver** [comment]
What about just using the original range, but then trying again it is invalid?

**Comment by kevintraver** [comment]
@rstacruz let me know your thoughts

**Comment by marocchino** [comment]
@kevintraver Thanks! It needs to clean up little bit but, good enough to merge :3

File: lib/ffaker/phone_number.rb [hunk 1]
```diff
@@ -30,7 +30,12 @@ def area_code
       # The North American Numbering Plan (NANP) does not permit the digits 0
       # and 1 as the leading digit of an area code.
       # https://en.wikipedia.org/wiki/List_of_North_American_Numbering_Plan_area_codes
-      rand(201..999)
+      rand_area_code = rand(201..999)
+      invalid_area_codes = [211,311,411,511,611,711,811,911]
+      while(invalid_area_codes.include?(rand_area_code))
+        rand_area_code = rand(201..999)
+      end
```
Comment (rstacruz) [inline lib/ffaker/phone_number.rb]: this works too:

```rb
do
  rand_area_code = ...
while invalid_area_codes.include?(rand_area_code)
```

Comment (marocchino) [inline lib/ffaker/phone_number.rb]: eda18d1 :3

**Pull request merged**